vaccine_feed_ingest/runners/us/va_gov/fetch.sh

```diff
@@ -10,14 +10,5 @@ else
     echo "Must pass an output_dir as first argument"
 fi
 
-# Per VA_gov spec available at https://developer.va.gov/explore/facilities/docs/facilities?version=current
-(cd "$output_dir" && curl -X GET 'https://sandbox-api.va.gov/services/va_facilities/v0/facilities/all' \
---header "apikey: ${TOKEN_FETCH_US_VA}" --create-dirs -o 'va_gov.json')
-
-
-
-##takes an API key that you can self-service create from https://developer.va.gov/apply -- or I can provide
-##this secret out-of-band via Discord or whatever
-
-##Note: Sandbox appears to be appropriate environment for read-only access; it appears prod is only for authorized developers that are building things that 
-##can change state on the VA server side.  Data is accurate as far as I can tell against what's available in real time on VA website.
+# Uses official VA API for searching for locations.  NOTE: underlying data source may not include some VA pop up vaccination events at smaller VA clinics or locations like VFW halls.
+(cd "$output_dir" && curl -X GET "https://api.va.gov/v1/facilities/va?bbox%5B%5D=-180&bbox%5B%5D=-90&bbox%5B%5D=180&bbox%5B%5D=90&type=health&services%5B%5D=Covid19Vaccine&page=1&per_page=2000&radius=25000&latitude=37.408123149415275&longitude=-93.14343299172322" --create-dirs -o 'va_gov.json')
```

Comment:
It looks like this currently returns `442` sites, which is well under the `2,000` site page size.

Reply:
> Thank you @davekasten and @nuchi! I think that this site should be a good fit for the [shared ndjson parser](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/JSON-to-NDJSON-Parser) to make the next step a bit easier!
> 
> One note: the script needs to be executable in order to run this via our tool. You can change this by running `chmod +x vaccine_feed_ingest/runners/us/va_gov/fetch.sh` and then committing the change.

Thanks -- I chmod'd the script and uploaded a new version.  Checking with ls -l returns that it has executable permissions, so I (think) it should be executable now